CODE_OF_CONDUCT.md

```diff
@@ -20,11 +20,13 @@ If you are being harassed, notice that someone else is being harassed, or have a
 
 We expect everyone to follow these rules anywhere in the CocoaPods project’s codebases, issue trackers, IRC channel, group chat, and mailing lists.
 
+This code of conduct applies both within project spaces and in public spaces when an individual is representing the project or its community.
```

Comment:
IANAL but I feel like a formulation like this would be less ambiguous:

> For [Core Contributors](http://cocoapods.org/about#team), who are considered to _represent the project or its community_ as individuals, this code of conduct is considered to be applied at all times, in public spaces as well as within project spaces, to ensure to show the value of a welcoming environment for anyone wanting to join.

What do you guys think?